assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/index.js

```diff
@@ -184,10 +184,11 @@ function DashboardAllTrafficWidget( {
 	} );
 
 	if ( entityURL ) {
```

Comment:
Should we maybe check `isURL` here instead of the nested conditions below?

Reply:
Yep, updated.